src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/JudgmentPaidInFullCallbackHandler.java

```diff
@@ -0,0 +1,113 @@
+            .put(callbackKey(ABOUT_TO_SUBMIT), this::saveJudgmentPaidInFullDetails)
+            .put(callbackKey(SUBMITTED), this::buildConfirmation)
+            .build();
+    }
+
+    private CallbackResponse buildConfirmation(CallbackParams callbackParams) {
+        return SubmittedCallbackResponse.builder()
+            .confirmationHeader(getHeader())
+            .confirmationBody(getBody())
+            .build();
+    }
+
+    private String getHeader() {
+        return format("# Judgment marked as paid in full");
+    }
+
+    private String getBody() {
+        return format("# Judgment marked as paid in full");
+    }
+
+    private CallbackResponse saveJudgmentPaidInFullDetails(CallbackParams callbackParams) {
+        CaseData caseData = callbackParams.getCaseData();
+        List<String> errors = new ArrayList<>();
+
+        if (nonNull(caseData.getJoOrderMadeDate())
+            && nonNull(caseData.getJoJudgmentPaidInFull().getDateOfFullPaymentMade())) {
+            boolean paidAfter30Days = JudgmentsOnlineHelper.checkIfDateDifferenceIsGreaterThan30Days(caseData.getJoOrderMadeDate(),
+                                                                              caseData.getJoJudgmentPaidInFull().getDateOfFullPaymentMade());
+            if (paidAfter30Days) {
+                caseData.getJoJudgmentStatusDetails().setJudgmentStatusTypes(JudgmentStatusType.SATISFIED);
+                if (caseData.getJoIsRegisteredWithRTL() == YesOrNo.YES) {
+                    caseData.getJoJudgmentStatusDetails().setJoRtlState(JudgmentsOnlineHelper.getRTLStatusBasedOnJudgementStatus(JudgmentStatusType.SATISFIED));
+                }
+            } else {
+                caseData.getJoJudgmentStatusDetails().setJudgmentStatusTypes(JudgmentStatusType.CANCELLED);
+                if (caseData.getJoIsRegisteredWithRTL() == YesOrNo.YES) {
+                    caseData.getJoJudgmentStatusDetails().setJoRtlState(JudgmentsOnlineHelper.getRTLStatusBasedOnJudgementStatus(JudgmentStatusType.CANCELLED));
+                }
+            }
+            caseData.getJoJudgmentStatusDetails().setLastUpdatedDate(LocalDateTime.now());
+            caseData.setJoIsLiveJudgmentExists(YesOrNo.NO);
+        }
+
+        CaseData.CaseDataBuilder<?, ?> caseDataBuilder = caseData.toBuilder();
+        return AboutToStartOrSubmitCallbackResponse.builder()
+            .data(caseDataBuilder.build().toMap(objectMapper))
+            .build();
+    }
+
+    private CallbackResponse validatePaymentDate(CallbackParams callbackParams) {
+        List<String> errors = new ArrayList<>();
+        CaseData caseData = callbackParams.getCaseData();
+        LocalDate dateOfPaymentMade = caseData.getJoJudgmentPaidInFull().getDateOfFullPaymentMade();
+
+        if (JudgmentsOnlineHelper.validateIfFutureDate(dateOfPaymentMade)) {
+            errors.add(String.format(ERROR_MESSAGE_DATE_MUST_BE_IN_PAST, ERROR_MESSAGE_DATE_MUST_BE_IN_PAST));
```

Comment:
I think String.format(ERROR_MESSAGE_DATE_MUST_BE_IN_PAST, ERROR_MESSAGE_DATE_MUST_BE_IN_PAST) is not required as there is no formating of string.

Reply:
Implemented the change